package-lock.json

```diff
@@ -5412,6 +5317,88 @@
         }
       }
     },
+    "phantomjs-prebuilt": {
+      "version": "2.1.14",
```

Comment:
Just checking, did you verify that this version works on earlier versions than OSX Sierra?